coreplugins/cesiumion/public/components/FormikErrorFocus.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { connect } from "formik";
+
+class FormikErrorFocus extends React.Component {
+    isObject(value) {
+        return (
+            value && typeof value === "object" && value.constructor === Object
+        );
+    }
+    getKeysRecursively = object => {
```

Comment:
Same thing with getKeysRecursively. Let's see if there is something built-in or another dependency we can use